src/Soql/ConnectionWrapper.php

```diff
@@ -121,13 +120,13 @@ public function update($tableExpression, array $data, array $identifier = [], ar
     private function addToBatchList(Request $request, array $refs): void
     {
         $command = [
-            'body' => json_decode($request->getBody()->getContents(), true),
+            'body'   => json_decode($request->getBody()->getContents(), assoc: true),
             'method' => $request->getMethod(),
-            'url' => (string) $request->getUri(),
+            'url'    => (string) $request->getUri(),
```

Comment:
Since we have one specific commit for `Allow new versions of guzzle and phpunit` should we also have one specific for the code style fix?